docs/docs-source/docs/modules/ROOT/pages/index.adoc

```diff
@@ -50,7 +50,7 @@ image::apps-1.png[]
 
 == Feature set
 
-Cloudflow includes tools for developing and deploying streaming applications:
+Cloudflow dramatically accelerates your application development efforts, reducing the time required to create, package, and deploy an application from weeks--to hours. Cloudflow includes tools for developing and deploying streaming applications:
```

Comment:
`development` / `developing` / `development` in 3 following sentences, it's not introduced in this PR, but would be nice to avoid the repetition.